src/core/Microsoft.PowerToys.Settings.UI/Views/ShellPage.xaml

```diff
@@ -92,6 +92,13 @@
                         <FontIcon Glyph="&#xEDA7;"/>
                     </winui:NavigationViewItem.Icon>
                 </winui:NavigationViewItem>
+
+                <!-- TO DO: Update icon -->
+                <winui:NavigationViewItem x:Uid="Shell_KeyboardManager" helpers:NavHelper.NavigateTo="views:KeyboardManagerPage">
+                    <winui:NavigationViewItem.Icon>
+                        <FontIcon Glyph="&#xEDA7;"/>
```

Comment:
This should be E765 as per #1752. Since it's just a single value change we can make it here rather than later.

Reply:
updated the icon